src/index.js

```diff
@@ -37,6 +37,9 @@ const Multiaddr = withIs.proto(function (addr) {
      */
     this.buffer = codec.fromBuffer(addr)
   } else if (typeof addr === 'string' || addr instanceof String) {
+    if (addr.length > 0 && addr.charAt(0) !== '/') {
+      throw new Error(`multiAddr "${addr}" must start with a "/"`)
```

Comment:
Last mile! Typo `multiAddr` should be `multiaddr`

Reply:
Fixed :)